docs/source/community_guides.rst

```diff
@@ -5,19 +5,19 @@
 Contributing
```

Comment:
To be consistent with all other pages, can we rename this page to Contributions

Reply:
Refactoring the community_guides.rst this point would cause this URL to change:
`https://ansible-collections.github.io/ibm_zos_core/community_guides.html` , without fair notice i don't know who might have linked to it, I recommend we take all refactors to the work item that is in place for the restructuring of the web docs site.